.eslintrc.js

```diff
@@ -6,16 +6,21 @@ module.exports = {
     node: true
   },
   extends: [
-    'eslint:recommended',
-    'plugin:css-modules/recommended',
-    'plugin:jsx-a11y/recommended',
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
     'plugin:prettier/recommended',
-    'plugin:react/recommended'
+    'plugin:css-modules/recommended',
+    'plugin:jsx-a11y/recommended'
   ],
   parserOptions: {
     ecmaVersion: 2019,
     sourceType: 'module'
   },
+  ecmaFeatures: {
+    jsx: true
+  },
+  parser: '@typescript-eslint/parser',
   plugins: ['css-modules', 'jsx-a11y', 'react', 'react-hooks'],
```

Comment:
Do we not also need `'@typescript-eslint'` in `plugins`? From https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration

Reply:
I suppose, I never noticed a difference.